tests/run_array_analyzer_test.py

```diff
@@ -1,3 +1,4 @@
+import argparse
 import os
 from testfixtures import TempDirectory
```

Comment:
in other repos I see package requirements for tests are separated from requirements for the main code base.  Do we want to do that here?  Currently testfixtures is not in requirements.txt

Reply:
@bryantChhun I agree that they shouldn't be in requirements. testfixtures and pytest etc are being installed during continuous integration (see continuous_integration.yml).
How about assuming developers can install necessary test packages locally for now and if we need more we can add a testing requirements later?

Reply:
@bryantChhun  and @jennyfolkesson  We do need to clean up requirements and test the pipeline like we did for Jetson nano. 

I found that xlrd was required on Jetson nano, but I don't recall which part required it.

If the unit tests require smaller set of packages, I agree it is better to
list and maintain those requirements separately to speed up the process.